tests/unit/amo/pages/TestCategoriesPage.js

```diff
@@ -53,4 +74,52 @@ describe(__filename, () => {
 
     expect(root.find(HeadLinks)).toHaveLength(1);
   });
+
+  it('returns a 404 when clientApp is Android and enableFeatureStaticThemesForAndroid is false', () => {
+    const { store } = dispatchClientMetadata({ clientApp: CLIENT_APP_ANDROID });
+    const _config = getFakeConfig({
+      enableFeatureStaticThemesForAndroid: false,
+    });
+
+    const root = renderThemeCategories({ _config, store });
+
+    expect(root.find(Categories)).toHaveLength(0);
+    expect(root.find(NotFound)).toHaveLength(1);
+  });
+
+  it('does not return a 404 when clientApp is Android and enableFeatureStaticThemesForAndroid is true', () => {
+    const { store } = dispatchClientMetadata({ clientApp: CLIENT_APP_ANDROID });
+    const _config = getFakeConfig({
+      enableFeatureStaticThemesForAndroid: true,
+    });
+
+    const root = renderThemeCategories({ _config, store });
+
+    expect(root.find(NotFound)).toHaveLength(0);
+    expect(root.find(Categories)).toHaveLength(1);
+  });
+
+  it('does not return a 404 when clientApp is not Android', () => {
+    const { store } = dispatchClientMetadata({ clientApp: CLIENT_APP_FIREFOX });
+    const _config = getFakeConfig({
+      enableFeatureStaticThemesForAndroid: false,
+    });
+
+    const root = renderThemeCategories({ _config, store });
+
+    expect(root.find(NotFound)).toHaveLength(0);
+    expect(root.find(Categories)).toHaveLength(1);
+  });
+
+  it('does not return a 404 when addonType is not "theme"', () => {
+    const { store } = dispatchClientMetadata({ clientApp: CLIENT_APP_ANDROID });
+    const _config = getFakeConfig({
+      enableFeatureStaticThemesForAndroid: false,
+    });
+
+    const root = render({ _config, store });
```

Comment:
I know extension is the default type (right?) but I think it would be a little clearer if you were to pass in the type here since that's the description for this test case.

Reply:
I did just notice that you did have a separate render for theme so maybe this is okay then..

Reply:
That's why I added `renderThemeCategories()` indeed.